src/apps/SlurCutter/gui/F0Widget.cpp

```diff
@@ -556,17 +621,17 @@ void F0Widget::paintEvent(QPaintEvent *event) {
                 painter.setPen(Qt::black);
                 painter.fillRect(rec, NoteColors[i.value.isSlur]);
                 painter.drawRect(rec);
-                if (i.value.ornament != OrnamentStyle::None) {
-                    if (i.value.ornament == OrnamentStyle::Up)
-                        noteDescText += "↗ ";
-                    if (i.value.ornament == OrnamentStyle::Down)
-                        noteDescText += "↘ ";
-                }
                 if (!std::isnan(i.value.cents)) {
                     painter.drawLine(rec.left(), rec.center().y(), rec.right(), rec.center().y());
                     noteDescText += PitchToNotePlusCentsString(i.value.pitch +
                                                 0.01 * (std::isnan(i.value.cents) ? 0 : i.value.cents));
                 }
+                if (i.value.glide != GlideStyle::None) {
+                    if (i.value.glide == GlideStyle::Up)
+                        noteDescText.prepend("↗");
+                    if (i.value.glide == GlideStyle::Down)
+                        noteDescText.append("↘");
+                }
```

Comment:
Adding comment about why there's such prepend/append difference would be better (the fundamentals of glides)

Reply:
Resolved in https://github.com/openvpi/dataset-tools/pull/3/commits/75032d164aedbbcf43bd8e9b7b9a054fbc390292